_posts/2024-11-06-mwinyi-tanzania-paper.md

```diff
@@ -0,0 +1,7 @@
+---
+layout: post
+title:  "Mwinyi et al. 2024 - A new cryptic taxon on the coast of East Africa"
```

Comment:
```suggestion
title:  "Mwinyi et al. 2024 preprint - A new cryptic taxon on the coast of East Africa"
```

Nit, I've been adding in "preprint" to the post title to help differentiate preprints from papers.